odp/compute_opt_traj.py

```diff
@@ -0,0 +1,127 @@
+from odp.Grid.GridProcessing import Grid
+# from first_order_generic import spa_deriv
+from update_state import next_state, update_state
+import numpy as np
+from find_tEarlist import find_sign_change
+
+def compute_opt_traj1v0(grid, V, tau, dynamics, x1_1v0, x2_1v0, subsamples=1):
```

Comment:
Please rename the function to compute_optTraj or something ( no 1v0, for general context). Also the variable x1_1v0 and x2_1v0